python/langsmith/evaluation/llm_evaluator.py

```diff
@@ -0,0 +1,285 @@
+"""Contains the LLMEvaluator class for building LLM-as-a-judge evaluators."""
+
+from typing import Any, Callable, Dict, List, Optional, Tuple, Union, cast
+
+from pydantic import BaseModel
+
+from langsmith.evaluation import EvaluationResult, EvaluationResults, RunEvaluator
+from langsmith.schemas import Example, Run
+
+
+class CategoricalScoreConfig(BaseModel):
+    """Configuration for a categorical score."""
+
+    key: str
+    choices: List[str]
+    description: str
+    include_explanation: bool = False
```

Comment:
I personally usually hyper-prompt engineer the explanation variable for my evals so don't love this being just a bool

Reply:
will have this be a list of attributes or dictionary where you can add descriptions